examples/all-clusters-app/esp32/sdkconfig.defaults.esp32c2

```diff
@@ -17,3 +17,6 @@ CONFIG_BT_NIMBLE_ROLE_OBSERVER=n
 CONFIG_ESP32_WIFI_STATIC_RX_BUFFER_NUM=4
 CONFIG_ESP32_WIFI_DYNAMIC_RX_BUFFER_NUM=8
 CONFIG_ESP32_WIFI_DYNAMIC_TX_BUFFER_NUM=16
+
+# Enable HKDF in mbedtls
+CONFIG_MBEDTLS_HKDF_C=y
```

Comment:
It's already in the sdkconfig.defaults, not necessary to add it to each SoC sdkconfig file

Reply:
Removed, but we should keep this config in `sdfconfig_xx.defaults` as it would not apply the configurations in `sdkconfig.defaults`